waku/v2/discv5/discover_test.go

```diff
@@ -22,6 +22,7 @@ import (
 )
 
 func createHost(t *testing.T) (host.Host, int, *ecdsa.PrivateKey) {
+	utils.InitLogger("console")
```

Comment:
Had to add this to avoid a panic, due to the logger change I introduced in our fork.